train_management/models.py

```diff
@@ -105,8 +105,8 @@ def __str__(self):
 class Train(models.Model):
 
     class Meta:
-        verbose_name = _("Train")
```

Comment:
Versuche bitte in einem PR nicht verschiedene Änderungen die nicht zusammenhängend sind zu vermeiden.